File: AndroidApp/MultiSportTrainer-Mobile/app/src/main/java/com/example/multisporttrainer/models/RegisterRequest.java
package com.example.multisporttrainer.models;

public class RegisterRequest {

    private String fullName;
    private String email;
    private String password;
    private String dateOfBirth;
    private String role;
    private String sportFocus;

    public RegisterRequest(String fullName, String email, String password) {
        this.fullName = fullName;
        this.email = email;
        this.password = password;
        this.dateOfBirth = null;
        this.role = "Player";
        this.sportFocus = "Football & Agility";
    }
}

File: Backend/MultiSportTrainerAPI/MultiSportTrainerAPI/DTOs/RegisterRequest.cs
namespace MultiSportTrainerAPI.DTOs
{
    public class RegisterRequest
    {
        public string FullName { get; set; } = string.Empty;

        public string Email { get; set; } = string.Empty;

        public string Password { get; set; } = string.Empty;

        public DateTime? DateOfBirth { get; set; }

        public string Role { get; set; } = "Player";

        public string? SportFocus { get; set; }
    }
}